docs/arqbackup-minio.md

```diff
@@ -0,0 +1,22 @@
+# Arq backup with Minio [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/minio/minio?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
+Arq backup software is available for Mac and Windows. Arq stores multiple versions of  files in encrypted format. The  backups can be stored on various cloud backends including Minio. This document will walk you through arqbackup configuration with Minio backend.   
+
+## 1. Prerequisites
+
+* [Arq](https://www.arqbackup.com/) backup installed.
+* Minio Server is running on localhost on port 9000 in ``HTTP``, follow [Minio quickstart guide](https://docs.minio.io/docs/minio-quickstart-guide) for installing Minio.
```

Comment:
Reword

* Minio Server is running on localhost:9000, follow [Minio quickstart guide] ...